index.js

```diff
@@ -24,7 +24,12 @@ module.exports = {
     "require-yield": "off",
 
     // Single quotes instead of double quotes
-    "quotes": ["error", "single", { "allowTemplateLiterals": true }],
+    "quotes": ["error", "single", {
+	"allowTemplateLiterals": true,
+
+	// Allow use of double quotes if it avoids escaping a single quote.
+	"avoidEscape": true
```

Comment:
my preference is to use backticks for strings that have single quotes.

Reply:
That works for me. It makes converting over to prettier slightly more work, but I think I have some ideas there.